rxandroid/build.gradle

```diff
@@ -1,11 +1,3 @@
-buildscript {
```

Comment:
Had to move to root `build.gradle` https://d.android.com/r/tools/buildscript-classpath-check.html